mobius-rx3/src/main/java/com/spotify/mobius/rx3/UnrecoverableIncomingException.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * -\-\-
+ * Mobius
+ * --
+ * Copyright (c) 2017-2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.mobius.rx3;
+
+/**
+ * Used to indicate that an {@link RxMobiusLoop} transformer has received an {@link
+ * io.reactivex.rxjava3.core.Observer#onError(Throwable)} call, which is illegal. This exception
+ * means Mobius is in an undefined state and should be considered a fatal programmer error. Do not
+ * try to handle this exception in your code, ensure it never gets thrown.
+ */
+class UnrecoverableIncomingException extends RuntimeException {
```

Comment:
Changed to package

Reply:
That should be fine, I think.

Reply:
the similar comment did not see usages but I can keep them public

Reply:
No, I meant, this exception should never be caught (as per the javadoc, even), so making it package-private is fine.

Reply:
haha OK, I got confused by the last `I think`